src/IonSpan.ts

```diff
@@ -164,6 +164,10 @@ export class StringSpan extends Span {
     return this._src.charCodeAt(ii);
   }
 
+  getCodePoint(index : number) : number {
+      return this._src.codePointAt(index);
```

Comment:
Does this take care of parsing the high/low surrogate pair into a singe code point?

Reply:
yes